--- "node_modules\\.pnpm\\helmet@5.1.0\\node_modules\\helmet\\dist\\types\\index.d.ts"	2022-06-28 00:53:22.576028800 +0300
+++ helmet.index.d.ts	2022-07-02 22:54:15.916937700 +0300
@@ -1,20 +1,20 @@
 /// <reference types="node" />
 import { IncomingMessage, ServerResponse } from "http"
-import contentSecurityPolicy, { ContentSecurityPolicyOptions } from "./middlewares/content-security-policy"
-import crossOriginEmbedderPolicy from "./middlewares/cross-origin-embedder-policy"
-import crossOriginOpenerPolicy, { CrossOriginOpenerPolicyOptions } from "./middlewares/cross-origin-opener-policy"
-import crossOriginResourcePolicy, { CrossOriginResourcePolicyOptions } from "./middlewares/cross-origin-resource-policy"
-import expectCt, { ExpectCtOptions } from "./middlewares/expect-ct"
-import originAgentCluster from "./middlewares/origin-agent-cluster"
-import referrerPolicy, { ReferrerPolicyOptions } from "./middlewares/referrer-policy"
-import strictTransportSecurity, { StrictTransportSecurityOptions } from "./middlewares/strict-transport-security"
-import xContentTypeOptions from "./middlewares/x-content-type-options"
-import xDnsPrefetchControl, { XDnsPrefetchControlOptions } from "./middlewares/x-dns-prefetch-control"
-import xDownloadOptions from "./middlewares/x-download-options"
-import xFrameOptions, { XFrameOptionsOptions } from "./middlewares/x-frame-options"
-import xPermittedCrossDomainPolicies, { XPermittedCrossDomainPoliciesOptions } from "./middlewares/x-permitted-cross-domain-policies"
-import xPoweredBy from "./middlewares/x-powered-by"
-import xXssProtection from "./middlewares/x-xss-protection"
+import contentSecurityPolicy, { ContentSecurityPolicyOptions } from "./middlewares/content-security-policy/index.js"
+import crossOriginEmbedderPolicy from "./middlewares/cross-origin-embedder-policy/index.js"
+import crossOriginOpenerPolicy, { CrossOriginOpenerPolicyOptions } from "./middlewares/cross-origin-opener-policy/index.js"
+import crossOriginResourcePolicy, { CrossOriginResourcePolicyOptions } from "./middlewares/cross-origin-resource-policy/index.js"
+import expectCt, { ExpectCtOptions } from "./middlewares/expect-ct/index.js"
+import originAgentCluster from "./middlewares/origin-agent-cluster/index.js"
+import referrerPolicy, { ReferrerPolicyOptions } from "./middlewares/referrer-policy/index.js"
+import strictTransportSecurity, { StrictTransportSecurityOptions } from "./middlewares/strict-transport-security/index.js"
+import xContentTypeOptions from "./middlewares/x-content-type-options/index.js"
+import xDnsPrefetchControl, { XDnsPrefetchControlOptions } from "./middlewares/x-dns-prefetch-control/index.js"
+import xDownloadOptions from "./middlewares/x-download-options/index.js"
+import xFrameOptions, { XFrameOptionsOptions } from "./middlewares/x-frame-options/index.js"
+import xPermittedCrossDomainPolicies, { XPermittedCrossDomainPoliciesOptions } from "./middlewares/x-permitted-cross-domain-policies/index.js"
+import xPoweredBy from "./middlewares/x-powered-by/index.js"
+import xXssProtection from "./middlewares/x-xss-protection/index.js"
 export interface HelmetOptions {
 	contentSecurityPolicy?: ContentSecurityPolicyOptions | boolean
 	crossOriginEmbedderPolicy?: boolean
